inst/example-equations.md

```diff
@@ -0,0 +1,9 @@
+Some text.
+
+$$
+E = m \times c^2
+$$
+
+How nice is this equation? Note that $\alpha=1$ and $b$ is not defined.
+
+$i_t = j_t$
```

Comment:
@xtimbeau I had added this.

Reply:
Great! I'll merge this if the checks pass. Then you can report new bugs in the coming weeks :smile_cat: